src/zjs_ocf.json

```diff
@@ -14,34 +15,27 @@
         "common": ["zjs_ocf_common.c"]
     },
     "zjs_config": ["-DBUILD_MODULE_OCF"],
+    "virtualdeps": ["net-l2"],
     "zephyr_conf": {
         "all": [
             "CONFIG_NETWORKING=y",
             "CONFIG_NET_IPV6=y",
             "CONFIG_NET_UDP=y",
             "CONFIG_TEST_RANDOM_GENERATOR=y",
             "CONFIG_INIT_STACKS=y",
-            "CONFIG_NET_NBUF_RX_COUNT=14",
-            "CONFIG_NET_NBUF_TX_COUNT=14",
+            "CONFIG_NET_BUF=y",
+            "CONFIG_NET_PKT_RX_COUNT=5",
```

Comment:
Can you tell me the reason you picked these numbers for OCF? 5, 2, 10, 6?  A working configuration?